src/DynamicData/List/ObservableListEx.cs

```diff
@@ -1713,7 +1713,7 @@ public static IObservable<IChangeSet<T>> FlattenBufferResult<T>(this IObservable
         /// <exception cref="System.ArgumentNullException">source</exception>
         public static IObservable<IChangeSet<T>> BufferIf<T>([NotNull] this IObservable<IChangeSet<T>> source,
             [NotNull] IObservable<bool> pauseIfTrueSelector,
-            bool intialPauseState = false,
```

Comment:
I suspect this is the same as #276 -- a tip for avoiding inclusion code in your pull requests is to use the git branch feature. So I tend to keep "master" matching the third party software version of master, then just branch off what I'm working on from there. For this PR it just means it should probably come after #276 is merged so we don't duplicate up.

Reply:
Yeah, I'll make sure to avoid cross pollination going forward

Reply:
I am happy for this one to go through as I do not foresee any issues